lib.php

```diff
@@ -24,6 +24,7 @@
 
 namespace tool_lifecycle\step;
 
+use tool_lifecycle\local\entity\process;
```

Comment:
is this being used? I only see $process, might be an accidental autostatement from IDE?

Reply:
Yes, in Line 113 in the documentation.

Reply:
Ah, I didn't consider it being used in the documentation